frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps {
+    children: React.ReactNode
+}
+
+export default function DropdownMenu( {children}: DropdownMenuProps) {
+    const [showDropdown, setShowDropdown] = useState(false);
+
+    function toggleDropdown() {
+        setShowDropdown(!showDropdown);
+    }
+
+    function hideDropdown(event: React.FocusEvent<HTMLButtonElement>): void {
+        if (event.currentTarget === event.target) {
+            setShowDropdown(false);
+        }
+    }
+
+    return (
+        <div className={showDropdown
```

Comment:
I read Andres comment and I do agree with it when a ternary becomes too long. This ternary is small enough to just keep on one line.

And if you want to get real fancy you could even write it like 
```<div className={`c-dropdon-menu${showDropdown ? '--active' : ''`}>``` 
but this would get close to personal preference if this is readable or not. I would not personally suggest it, just show you an alternative :)